### DISCUSSION TITLE
lookup-service-level-1 Publish test results to Alva

### DISCUSSION THREAD

**Discussion by Polyakovleo** [DevSkillsHQ/discusssions#83] [Help] [answered]
Hi team, 
I completed my coding test using the GitHub Codespace developer environment. I finished test 5 of 5 and submitted a pull request to GitHub. However, when I tried to publish the test results, it showed a failure: 2 tests passed, and 3 tests failed.
![Capture](https://github.com/DevSkillsHQ/discusssions/assets/20354647/b64ac502-fedf-4ac1-83c1-6108eaff9d61)
Please help.

**Comment by Ostter** [comment]
Hej, [Polyakovleo](https://github.com/Polyakovleo)
I checked locally and in the codespace and got the same failed test. I guess the problem may be with your implementation, and I recommend you check again.
One note that after initializing a boilerplate, you have to work only in the app folder, and should not change other files 
Try to put all changes Lookup.js to the app folder app/server/app.js

**Comment by Polyakovleo** [comment]
Hi Konstantin,
Thanks for clarification, next time I will do the solution in the app folder.

I created codebase and in the GitHub Codespace I create solution. 
And I followed instructions here(https://help.alvalabs.io/en/articles/9028831-how-to-work-with-cypress-tests)
(Running the tests using the terminal)
ran server, and in second terminal ran 
`npm i && npm run test `
and got 5 of 5 test result. And based on this result I think implementation was done.
Could you check if it works for you also.
Thanks in advance.

**Comment by ehsan-alva** [comment]
Hello @Polyakovleo,

To successfully pass the tests, please ensure that your solution is implemented within the `/app` folder. If you are using any boilerplates, avoid making changes outside this folder.

**Reply by Polyakovleo** [reply]
Hi Ehsan, I create new branch, added my solution to /app folder as you suggested. 
Make internal test as was described above, got 5 of 5 test passed 
![image](https://github.com/DevSkillsHQ/discusssions/assets/20354647/971fb6e5-a6f9-4fcf-a2e6-d873f6fd41e9)
Add all changed files, Commit an pull request. But have the same error 2 test passed 3 failed.
Please could you help me to figure out the issue. 
Thanks.

**Reply by Polyakovleo** [reply]
In the logs you can see server works and response. Point 1 and 2
and 404 for any request for data Point 3
![image](https://github.com/DevSkillsHQ/discusssions/assets/20354647/48842a29-7135-4816-8dd4-1653885e4100)
in the task description was pointed the address for fetching data. I can test it on my side and on  GitHub Codespace as  you can see above all tests passed.
Only when I publish it automatic test show fails. This is very simple coding task, I checked it twice. 
P.S.
I really want that job position at the company, which request it. I would like to show good result.
Thanks in advance.

**Reply by ehsan-alva** [reply, marked as answer]
@Polyakovleo, please note that in the package.json, the app initiates by executing "nodemon ./src/server/index.js", which differs from the path used in your implementation. This is a crucial detail that forms part of the overall challenge.

**Reply by Polyakovleo** [reply]
Hi @ehsan-alva Thanks for help.
It was really point of problem.
